packages/server/modules/workspaces/graph/resolvers/workspaces.ts

```diff
@@ -512,6 +537,26 @@ export = FF_WORKSPACES_MODULE_ENABLED
 
           const invite = await ctx.loaders.invites.getInvite.load(inviteId)
           return invite?.token || null
+        },
+        email: async (parent, _args, ctx) => {
+          if (!parent.user) return parent.email
+
+          // TODO: Tests to check token & email access?
```

Comment:
Need to add tests for all of the changed backend logic, but considering the lack of time I'd like to approach this after